### PR TITLE
Update submodule versions.

### PR DESCRIPTION
**Pull request by brianhuffman**
Update stack.yaml files to adapt:
 ghc-8.4 was dropped in GaloisInc/cryptol#789.
 SBV version bound was updated in GaloisInc/cryptol@1be435c6.

### CODE REVIEW

**Comment by brianhuffman** [comment]
The travis build is failing. It looks like it's trying to run the saw-script integration tests before it has compiled `saw`, so it's printing out error messages like `test.sh: 1: eval: saw: not found` for something like 70 different tests.

File: stack.ghc-8.6.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ extra-deps:
   - splitmix-0.0.4
   - bv-sized-1.0.1
   - bitwise-1.0.0.1
-  - sbv-8.4
+  - sbv-8.7
```
Comment (robdockins) [inline stack.ghc-8.6.yaml:50]: FYI, SBV 8.7 requires a _quite_ recent version of CVC4, and the bug I originally pulled in SBV 8.7 for has now been fixed a different way.

Comment (atomb) [reply stack.ghc-8.6.yaml:50]: Should we fall back on 8.6, then?

Comment (brianhuffman) [reply stack.ghc-8.6.yaml:50]: So maybe I should relax this to 8.6?

Comment (brianhuffman) [reply stack.ghc-8.6.yaml:50]: Well, I guess in a `stack.yaml` file it's not "relaxing" the version number, as it's not a lower bound. Maybe I should "downgrade" it to 8.6?

Comment (brianhuffman) [reply stack.ghc-8.6.yaml:50]: Actually the ghc-8.8.yaml file has sbv-8.6, so it would be good for those to match.

Comment (robdockins) [reply stack.ghc-8.6.yaml:50]: I think that's probably the right course of action for now.

**Pull request merged**